src/Sylius/Component/Channel/Factory/ChannelFactoryInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Channel\Factory;
+
+use Sylius\Component\Channel\Model\ChannelInterface;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+interface ChannelFactoryInterface extends FactoryInterface
```

Comment:
By extending `FactoryInterface` we couple Channel component with Resource bundle.